yarn.lock

```diff
@@ -2402,7 +2402,7 @@
     safe-flat "^2.0.2"
 
 "@okta/okta-signin-widget@link:dist":
-  version "7.6.0"
+  version "7.7.0"
```

Comment:
This happened just on a `yarn install` before I made any changes to resolutions.